interface/src/WaveformVisualizer.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useRef, useEffect } from "react";
```

Comment:
Not sure how the UI looks like at the moment (waiting for your demo video) - but I don't think this view is necessary

Reply:
This component is the waveform visualization of the audio data, just like in the original application. Here's a quick picture:
<img width="1680" alt="Screenshot 2023-07-27 at 6 35 36 PM" src="https://github.com/saharmor/whisper-playground/assets/79014814/0c3f3271-0cc9-4795-87c6-faef04c67d2f">